plugins/LilyPad/LilyPad.cpp

```diff
@@ -1224,6 +1228,23 @@ u8 CALLBACK PADpoll(u8 value)
                     Update(query.port, query.slot);
                     ButtonSum *sum = &pad->sum;
 
+                    if (config.padConfigs[query.port][query.slot].type == MousePad) {
+                        u8 b1 = 0xFC;
+                        if (sum->buttons[5 + 4] > 0) // Left button
+                            b1 -= 2 << 2;
+                        if (sum->buttons[6 + 4] > 0) // Right button
+                            b1 -= 2 << 1;
```

Comment:
It was more in line with the other digital button entries, but it's probably more confusing this way with the way it's used, so I've simplified it.